service/controller/resource/s3object/desired_test.go

```diff
@@ -52,9 +52,8 @@ func Test_DesiredState(t *testing.T) {
 	}
 	clientset := fake.NewSimpleClientset(release)
 
-	masterKeyPattern := "cloudconfig/v[\\d_]+/master"
-	workerKeyPattern := "cloudconfig/v[\\d_]+/worker"
-
+	masterKeyPattern := "ignition/master"
```

Comment:
I am not sure on the impact this could have during upgrades. The source for the ignition of old workers would change - this should not be an issue, but we should make sure that it isn't.